components/kyma-environment-broker/internal/process/provisioning/event_hub_step_test.go

```diff
@@ -434,6 +444,22 @@ func fixProvisioningOperation(t *testing.T) internal.ProvisioningOperation {
 	return op
 }
 
+func fixProvisioningOperationWithInvalidName(t *testing.T) internal.ProvisioningOperation {
+	op := internal.ProvisioningOperation{
+		Operation: internal.Operation{},
+		ProvisioningParameters: `{
+			"plan_id": "4deee563-e5ec-4731-b9b1-53b42d855f0c",
+			"parameters": {
+                "name": "foo",
+        		"components": [],
+				"region": "westeurope"
```

Comment:
(nit) indention is wrong:

```suggestion
			"parameters": {
				"name": "foo",
				"components": [],
				"region": "westeurope"
```

Reply:
Fixed